<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherMainWindow</class>
 <widget class="QMainWindow" name="TeacherMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="standardRecordButton">
      <property name="text">
       <string>Standard Fingering Record</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>&amp;Exit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="dataViewerButton">
      <property name="text">
       <string>Students' Data View</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="addUserButton">
      <property name="text">
       <string>Add a User</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <layout class="QGridLayout" name="poseLayout"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="tmplMgrButton">
      <property name="text">
       <string>Fingering Template Manger</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
